py/README.md

```diff
@@ -1,3 +1,25 @@
 # Terra widgets
 
 A python package for ipywidget-based user interfaces for performing tasks within the context Python Jupyter notebooks running in either the Terra or All of Us workbench environments.
+
+
+## Create and view HTML snapshots of notebooks
+
+The workbench takes care of saving the current version of your notebooks for you. But what if you want to know **what your notebook looked like two weeks ago?** Use `display_html_snapshots_widget()` to display a user interface that allows users to track how results in our notebooks change over time. To do this, it:
```

Comment:
nit: on first read it sounds to me like you'll be able to use this to look into the past, perhaps despite never having used this package before. Maybe just a slight rephrasing here, something like:

```
... display a widget which can save snapshots of a notebook for later review, allowing users to track changes in notebooks over time.
```

Reply:
done, thx!